enable/tests/test_assistant_test_case.py

```diff
@@ -1,7 +1,4 @@
-try:
-    from unittest import mock
-except ImportError:
-    import mock
+from unittest import mock
 import nose
```

Comment:
Note: `unitest` is std library and `nose` is 3rd party, so there should be a blank line between these imports.